.github/workflows/ci.yaml

```
@@ -0,0 +1,46 @@
+          echo "Folders by path: ${{ steps.packages.outputs.folders_by_path }}"
+  aur:
+    name: ${{ matrix.package }}
+    needs: [packages-list]
+    runs-on: ubuntu-latest
+    environment: ${{ matrix.package }}
+    defaults:
+      run:
+        working-directory: ${{ matrix.package }}
+    strategy:
+      matrix:
+        package: ${{ needs.packages-list.outputs.packages }}
```

actionlint (1.7.9)

type of expression at "matrix row" must be array but found type string

(expression)